backend/src/main/resources/demo/db/migration/V0004__user_insert.sql

```diff
@@ -1 +1 @@
-INSERT INTO guser (id, login, password) VALUES(1, 'admin', 's5wqy52qDD53eL6mVUFqiAuruW/wgpo5H9wv3IdDqz7WqyPjFOaxEQ==');
\ No newline at end of file
+INSERT INTO guser (id, login, password, admin) VALUES(1, 'admin', 's5wqy52qDD53eL6mVUFqiAuruW/wgpo5H9wv3IdDqz7WqyPjFOaxEQ==', true);
```

Comment:
This file was right from the start. When starting from an empty database, this file is run before the admin flag exists in the schema, so that SQL would break.
The flag is introduced in the next revision V0005__admin_flag, and it sets the admin flag to true for existing users (i.e. only admin at this time).